src/main.ts

```diff
@@ -196,4 +200,55 @@ async function addWorkflowSummary(msg: Message): Promise<void> {
   await summary.write();
 }
 
+async function sendSlackNotifications(
+  inputs: Inputs,
+  msg?: Message,
+): Promise<void> {
+  if (!inputs.slackWebhook) {
+    core.debug("Skipping Slack notification (no webhook)");
+    return;
+  }
+
+  // Use presence of msg to know if we're sending failure or success
+  const { color, title, value } = msg
+    ? {
+        color: "failure",
+        title: `CI/CD on ${github.context.repo} has been halted`,
+        value: message.toString(msg),
```

Comment:
It might be nice to check if this `=== DEFAULT_TITLE` (i.e. there was actually no message in the file) and not add anything in that case. Otherwise it'll be a bit redundant.